storage/client/client.go

```diff
@@ -844,10 +844,19 @@ func (c *StorageClient) Account(ctx context.Context, address staking.Address) (*
 }
 
 // Computes shares worth given total shares and total balance.
-func amountFromShares(shares common.BigInt, totalShares common.BigInt, totalBalance common.BigInt) common.BigInt {
+func amountFromShares(shares common.BigInt, totalShares common.BigInt, totalBalance common.BigInt) (common.BigInt, error) {
+	if shares.IsZero() {
```

Comment:
:information_source: db has some leftover cases where shares=0 and totalShares=0. this prevents us from complaining in that case

(is that right? in my understanding from the PR description and related PRs)

Reply:
> is that right? in my understanding from the PR description and related PRs

Yes, although I'm not 100% sure if shares=0 and totalShares=0 will still exist after all PRs are deployed and we do a reindex.

Reply:
Will recheck in future